lldb/source/Breakpoint/BreakpointResolverScripted.cpp

```diff
@@ -50,8 +50,28 @@ void BreakpointResolverScripted::CreateImplementationIfNeeded(
   if (!script_interp)
     return;
 
-  m_implementation_sp = script_interp->CreateScriptedBreakpointResolver(
-      m_class_name.c_str(), m_args, breakpoint_sp);
+  m_interface_sp = script_interp->CreateScriptedBreakpointInterface();
+  if (!m_interface_sp) {
+    m_error = Status::FromErrorStringWithFormat(
+        "BreakpointResolverScripted::%s () - ERROR: %s", __FUNCTION__,
+        "Script interpreter couldn't create Scripted Breakpoint Interface");
+    return;
+  }
+  
+  auto obj_or_err = m_interface_sp->CreatePluginObject(
+      m_class_name, breakpoint_sp, m_args);
+  if (!obj_or_err) {
+    m_error = Status::FromError(obj_or_err.takeError());
+    printf("CreateImplementationIfNeeded got error: %s\n", m_error.AsCString());
```

Comment:
```suggestion
```

Reply:
https://github.com/llvm/llvm-project/issues/151045

Reply:
I just said "we should come up with a good scheme" I didn't opine on the scheme.